src/components/switch/switch.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import styles from './switch-styles.scss';
+
+class Switch extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { selectedOption: props.selectedOption };
+  }
+
+  handleOnClick(option) {
+    const { onClick } = this.props;
+    this.setState({ selectedOption: option.value });
+    onClick();
```

Comment:
I think it will be useful to know what option you selected in the callback, so I'd probably send it in the callback prop too.

Reply:
👍